src/common/functions/calculateAverage.ts

```diff
@@ -0,0 +1,4 @@
+export default function calculateAverage(array: number[]) : string {
+  const total = array.reduce((total: number, item: number) => total + item, 0);
+  return (total / array.length).toFixed(2);
```

Comment:
Good catch! This will make the function more generic as well. Updated the code at [49f6d2f](https://github.com/AfterClass-io/afterclass.io-v2/pull/133/commits/49f6d2f6aef724fd7bae08435bfe37f2288054d9)